src/api/like/content-types/like/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "kind": "collectionType",
+  "collectionName": "likes",
+  "info": {
+    "singularName": "like",
+    "pluralName": "likes",
+    "displayName": "Like",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "objectId": {
+      "type": "string"
+    },
+    "objectType": {
+      "type": "string"
+    },
+    "userId": {
+      "type": "string"
```

Comment:
We can use a relation field linking to the users collection, like https://github.com/dev-launchers/strapiv4/blob/main/src/api/idea-card/content-types/idea-card/schema.json#L53.

Reply:
Thanks for the feedback and examples here. I have addressed all 3 with my latest commit @chungthuang